report_thesis/src/sections/multi-task-learning.tex

```diff
@@ -0,0 +1,8 @@
+In their work \citet{caruana_no_1997} presents a method called \textit{Multitask learning} which is a method of learning machine models on several related datasets. 
+The motivation for this approach stems from the assumption that utilizing multiple, albeit related, datasets can enhance the model's ability to discern patterns and shapes within the data.
```

Comment:
```suggestion
The motivation for this approach stems from the assumption that utilizing multiple, albeit related, datasets can enhance a model's ability to discern patterns and shapes within the data.
```
Should probably be unspecific since we are talking about models in general.